python/nx-cugraph/README.md

```diff
@@ -89,48 +89,106 @@ interface to its CUDA-based graph analytics library) and
 [CuPy](https://cupy.dev/) (a GPU-accelerated array library) to NetworkX's
 familiar and easy-to-use API.
 
-Below is the list of algorithms (many listed using pylibcugraph names),
-available today in pylibcugraph or implemented using CuPy, that are or will be
+Below is the list of algorithms that are currently supported or planned to be
```

Comment:
I think it's more accurate to say _functions_, not algorithms here (and in the section header).